static/scss/resource.scss

```
@@ -5,11 +5,172 @@ html {
 }
 
 .resource-container {
-  padding-top: 30px;
   padding-bottom: 30px;
   color: #000000;
   font-size: 16px;
   line-height: 28px;
+  position: relative;
+
+  .banner-holder {
+    position: relative;
+    height: 270px;
+
+    @include media-breakpoint-down(sm) {
+      height: 200px;
+    }
+
+    img {
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  .resource-content {
+    overflow: hidden;
+    padding: 70px 30px 15px;
+    position: relative;
+    z-index: 1;
+    margin-top: -60px;
+
+    @include media-breakpoint-down(sm) {
+      margin-top: -30px;
+      padding: 30px 15px 15px;
+    }
+
+    &:after {
+      position: absolute;
+      left: 200px;
+      top: 0;
+      right: 0;
+      height: 100%;
+      background: $white;
+      content: "";
+
+      @include media-breakpoint-down(sm) {
+        left: 5px;
+        right: 5px;
+      }
+    }
+
+    &.singal-child {
```

perhaps this should be `single-child` ? 

```suggestion
    &.single-child {
```